2022 - Legiones Astartes.cat

```diff
@@ -24456,13 +24456,552 @@ A Perfect Fury: If a model or models with the Skill Unmatched special rule selec
+                <modifier type="decrement" field="f111-2ce5-dd12-d6b0" value="1">
+                  <conditionGroups>
+                    <conditionGroup type="or">
+                      <conditionGroups>
+                        <conditionGroup type="and">
+                          <conditions>
+                            <condition field="selections" scope="ac2d-77a1-ecd7-a555" value="1.0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" childId="4af0-7de7-45cc-6565" type="notEqualTo"/>
+                            <condition field="selections" scope="ac2d-77a1-ecd7-a555" value="1.0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" childId="89fa-96fc-fa4a-eddb" type="notEqualTo"/>
+                          </conditions>
+                        </conditionGroup>
+                        <conditionGroup type="and">
+                          <conditions>
+                            <condition field="selections" scope="ac2d-77a1-ecd7-a555" value="1.0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" childId="4266-4404-9224-8a49" type="notEqualTo"/>
+                            <condition field="selections" scope="ac2d-77a1-ecd7-a555" value="1.0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" childId="f890-209d-317b-5b44" type="notEqualTo"/>
+                          </conditions>
+                        </conditionGroup>
+                      </conditionGroups>
+                    </conditionGroup>
+                  </conditionGroups>
+                </modifier>
+              </modifiers>
+              <characteristics>
+                <characteristic name="Unit Type" typeId="ddd7-6f5c-a939-b69e">Dreadnought (Heavy)</characteristic>
+                <characteristic name="Move" typeId="893e-2d76-8f04-44e5">6</characteristic>
+                <characteristic name="WS" typeId="cc42-7ed5-7092-5c84">5</characteristic>
+                <characteristic name="BS" typeId="74ae-c840-0036-d244">5</characteristic>
+                <characteristic name="S" typeId="e478-41d4-a092-48a8">8</characteristic>
+                <characteristic name="T" typeId="c32b-5fdd-3fbe-9b1f">8</characteristic>
+                <characteristic name="W" typeId="57ee-1126-32a9-5672">7</characteristic>
+                <characteristic name="I" typeId="62d3-22d7-2d49-52dc">4</characteristic>
+                <characteristic name="A" typeId="f111-2ce5-dd12-d6b0">5</characteristic>
+                <characteristic name="Ld" typeId="e8a6-1da9-d384-8727">9</characteristic>
+                <characteristic name="Save" typeId="e593-6b3c-f169-04f0">2</characteristic>
+              </characteristics>
+            </profile>
+          </profiles>
+          <rules>
+            <rule id="c2d0-9f6b-7652-c785" name="Move Through Cover" publicationId="e77a-823a-da94-16b9" page="243" hidden="false">
```

Comment:
These should still be rules links, unless they are specific only to the leviathan. I know at least hammer of wrath and dreadnought talon are.